app/controllers/events_controller.rb

```diff
@@ -8,8 +8,14 @@ def index
       geocoded_search_results = Geocoder.search(params[:query])
       top_result = geocoded_search_results.first
       @events = policy_scope(Event).near(top_result.address, 5)
+      @events_sorted = @events
     else
-      @events = policy_scope(Event)
+      @events = policy_scope(Event).where.not(latitude: nil)
```

Comment:
This line is really cool, was it part of the lecture or did you find this online/asking?

Reply:
Asking Google "active record where not nil". Active record is pretty cool!